src/gui/folderwizard.cpp

```diff
@@ -482,9 +482,8 @@ void FolderWizardRemotePath::showWarn( const QString& msg ) const
 FolderWizardSelectiveSync::FolderWizardSelectiveSync(const AccountPtr& account)
 {
     QVBoxLayout *layout = new QVBoxLayout(this);
-    _treeView = new SelectiveSyncTreeView(account, this);
-    layout->addWidget(new QLabel(tr("Choose What to Sync: You can optionally deselect remote subfolders you do not wish to synchronize.")));
```

Comment:
The new widget is (header label, tree view, big folder notice (if necessary)). The new header text is going to be slightly less specific than before since it always says the same thing. But I think it's okay and nicely unifies some more aspects of selective sync setup.